src/components/Validate/index.tsx

```diff
@@ -191,9 +189,9 @@ const Validate = ({ toggleSnackbar, setSnackbarMessage, setLoading }: IHomeProps
           >
             <Typography variant='h5'>{`subject: ${subjectValue}`}</Typography>
             <Typography variant='h5'>{`claim: ${claimVerbValue}`}</Typography>
-            <Typography variant='h5'>{`object: ${objectValue}`}</Typography>
-            <Typography variant='h5'>{`amt: ${amtValue}`}</Typography>
-            <Typography variant='h5'>{`effectiveDate: ${effectiveDateValue}`}</Typography>
+            {objectValue && <Typography variant='h5'>{`object: ${objectValue}`}</Typography>}
+            {amtValue && <Typography variant='h5'>{`amt: ${amtValue}`}</Typography>}
+            {amtValue && <Typography variant='h5'>{`effectiveDate: ${amtValue}`}</Typography>}
```

Comment:
Amr, you add amt value to the effective date, please fix it

Reply:
Yeh I know but forget to replace it thanks for letting me know